Loop/Models/RemoteCommand.swift

```diff
@@ -70,10 +70,27 @@ extension RemoteCommand {
                 absorptionTime = TimeInterval(hours: absorptionOverride)
             }
             let quantity = HKQuantity(unit: .gram(), doubleValue: carbsValue)
-            let newEntry = NewCarbEntry(quantity: quantity, startDate: Date(), foodType: "", absorptionTime: absorptionTime)
+            
```

Comment:
Parse the created-date. I'm using the same date format as used for the expire date in notifications. 

I'm making this field optional for backwards compatibility with NS versions that don't pass this value. But when it is there, this expects it to have the right format.